SpatialGDK/Extras/schema/rpc_components.schema

```diff
@@ -12,14 +12,16 @@ type Void {
 
 component UnrealClientRPCEndpoint {
     id = 9990;
-    event UnrealRPCPayload unreliable_client_to_server_rpc;
-    command Void reliable_server_to_client_rpc(UnrealRPCPayload);
+    bool ready = 1;
+    event UnrealRPCPayload client_to_server_rpc_event;
+    command Void server_to_client_rpc_command(UnrealRPCPayload);
```

Comment:
If this is also being used for cross server then this name isn't accurate.

Reply:
I like that current naming tells the direction of events and commands. When it's cross-server - we could still think of one server being a client in some context to another server? Otherwise, what name would you prefer?